go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/dapr/components-contrib v1.15.6
 	github.com/dapr/durabletask-go v0.6.8
 	github.com/dapr/kit v0.15.4
-	github.com/diagridio/go-etcd-cron v0.8.3
+	github.com/diagridio/go-etcd-cron v0.9.1
```

Comment:
Please add release notes.

Reply:
You mean in the issue description? I checked both https://github.com/dapr/dapr/pull/9003 and https://github.com/dapr/dapr/pull/9015 and they don't have release notes.

Reply:
@acroca this is part of a backport which I assume will be a patch release so needs a new release notes for v1.15.12 - https://github.com/dapr/dapr/tree/master/docs/release_notes

Reply:
Added those, thanks